c10/util/complex_math.h

```diff
@@ -290,7 +290,7 @@ C10_HOST_DEVICE inline c10::complex<T> atanh(const c10::complex<T>& x) {
 }
 
 template <typename T>
-C10_HOST_DEVICE inline c10::complex<T> log1p(const c10::complex<T>& z) {
+C10_HOST_DEVICE c10::complex<T> log1p(const c10::complex<T>& z) {
```

Comment:
This inline keyword isn't an inlining hint, but to ensure an identical function exists in every translation unit (which is usually desirable behavior). Removing this could cause an violation of the ODR rule, which is undesirable.

Reply:
Ah, I see. Thanks. I'll restore the `inline` in the next commit. My experiment also shows that removing this does not have a significant effect.